CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-03-31
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* Support upgrade version skew policy between core node and control plane components from n-2 to n-3 to match [related upstream policy change starting Kubernetes version 1.28](https://kubernetes.io/blog/2023/08/15/kubernetes-v1-28-release/) AKS docs available [here](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#what-is-the-allowed-difference-in-versions-between-control-plane-and-node-pool).
+* Starting 1.30 Kubernetes version and 1.27 LTS versions, beta apis will be disabled by default, when you upgrade to [them](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli#before-you-begin). There will be an option provided to explicitly enable beta apis closer to the 1.30 release.
+* If you use any programming or scripting logic to list and  select minor version before creating clusters - please ensure you update the logic to exclude LTS-only versions to avoid any breaks.  Otherwise, if LTS is your path forward please first opt into premium tier and LTS support plan as mentioned in [Long-term support for Azure Kubernetes Service (AKS) - Azure Kubernetes Service | Microsoft](https://learn.microsoft.com/azure/aks/long-term-support) before the create operation.
```

Comment:
Not sure if my previous comment landed or not, but this is still unclear on what is the bad case and what is the correct case customers should do. Also don't see a doc link that contains such info. @kaarthis @haitch

Reply:
i ll start a GH issue first and we can clarify more there then follow up with a doc.  We can start populating from April rel. notes onward.  Suggest you can remove it from this one @abubinski

Reply:
Okay removed, thanks.